campaigns/rustreach/how-I-came-to-rust.md

```diff
@@ -0,0 +1,58 @@
+---
+title: How I came to Rust
+---
+
+# How I came to Rust
+
+The #rustreach program asked people to write down how they came to Rust and I'm happy to do that. Entering a community or a space is always a moment of orientation process and I hope I can give some insight into that.
+
+My path is highly individual, but yours will be, too. Own that. It's great.
+
+## Picking up Rust
+
+I am certainly an early adopter of Rust. I picked it up as a hobby in the early versions, Rust 0.4 was the first version I tried out and around version 0.6, I started writing notable pieces of software in it. That was in 2013. Rust - for me - was the chance to see a language being built. I had no interested in writing a programming language, but I was curious how the process works. I'm still amazsed on how agressively languages can change when compatibilty is no issue. Back then, _all Rust code in existence_ would break _regularly_.
+
+But there were two things I found very notable:
+
+* The openness of the discussion and the ease of participation
+* The air around the community to build something to _use_
+
+Rust was always the attempt to apply techniques instead of coming up with their own (obviously, the process means that you need to come up with your own things). It was interesting to watch the process and rare to see that. I enjoyed watching and coding along.
```

Comment:
I'm not sure I quite understood what this bit means about "applying techniques instead of coming up with their own"? Is it talking about the designers of the language? I thought from the earlier paragraph that it might be more like the opposite: because it is a new language, like an open book, it would be possible to come up with new suggestions rather than having to use existing techniques? Perhaps clarifying this slightly or adding an example might help?